docs/getting-started-guides/kubeadm.md

```diff
@@ -173,16 +173,48 @@ This will remove the "dedicated" taint from any nodes that have it, including th
 
 ### (3/4) Installing a pod network
 
-You must install a pod network add-on so that your pods can communicate with each other.
-
-**It is necessary to do this before you try to deploy any applications to your cluster, and before `kube-dns` will start up. Note also that `kubeadm` only supports CNI based networks and therefore kubenet based networks will not work.**
-
-Several projects provide Kubernetes pod networks using CNI, some of which 
-also support [Network Policy](/docs/user-guide/networkpolicies/). See the [add-ons page](/docs/admin/addons/) for a complete list of available network add-ons.
-
-You can install a pod network add-on with the following command: 
-
-    # kubectl apply -f <add-on.yaml>
+You must install a pod network add-on so that your pods can communicate with each other, a cluster without a network add-on should not be considered fully-functional.
+
+**It is necessary to do this before you try to deploy any applications to your cluster.**
+
+
+Several projects provide Kubernetes pod networks using CNI, some of which also support [Network Policy](/docs/user-guide/networkpolicies/).
+The dropdown below includes some common network add-ons. Select one from the dropdown to view its installation instructions.
+
+See the [add-ons page](/docs/admin/addons/) to learn more about each of the available network add-ons.
+
+<script>
+  var networkAddons = {
+    "Calico": "kubectl apply -f \"http://docs.projectcalico.org/v2.0/getting-started/kubernetes/installation/hosted/kubeadm/calico.yaml\"",
+    "Flannel": "kubectl apply -f \"https://github.com/coreos/flannel/blob/master/Documentation/kube-flannel.yml?raw=true\"",
```

Comment:
Flannel also needs to apply a RBAC file to run properly: https://github.com/coreos/flannel/blob/master/Documentation/kube-flannel-rbac.yml
Would be helpful to have a single file a no external commands.